spec/Candid.md

```diff
@@ -1024,12 +1058,12 @@ The relations above have certain properties. As in the previous section, `<v> :
   ```
   does not imply
   ```
-  C[<t1> <: <t3>] = C[<t2> <: <t3>] ⚬ C[<t1> <: <t2>]
+  C[<t1> ~> <t3>] = C[<t2> ~> <t3>] ⚬ C[<t1> ~> <t2>]
```

Comment:
How to formulate this part?

Reply:
We can actually drop the assumption about subtyping, and state

NB: Transitive coherence does not hold:
```
<v1> : <t1> ~> <v2> : <t2>,  <v2> : <t2> ~> <v3> : <t3>, 
```
and
```
<v1> : <t1> ~> <v3'> : <t3> 
```
do not imply `<v3> = <v3'>`.

However, it implies `<v3> ~ <v3'>` is the smallest homomorphic, reflexive, symmetric relation that satisfies `∀ v. opt v ~ null`.